us.cityofnewyork.data/ems-incident-dispatch/daylight_savings_effect.ipynb

```diff
@@ -26,7 +26,39 @@
    "source": [
     "%%sql\n",
     "CREATE OR REPLACE TEMPORARY VIEW dispatch_serious AS (\n",
-    "    SELECT * FROM dispatch\n",
+    "    SELECT\n",
```

Comment:
Was `select *` somehow problematic?

Reply:
Yes, because of `offset` column presence.